src/fondant/data_io.py

```diff
@@ -160,7 +158,11 @@ def __init__(
     ):
         super().__init__(manifest=manifest, component_spec=component_spec)
 
-    def write_dataframe(self, dataframe: dd.DataFrame) -> None:
+    def write_dataframe(
+        self,
+        dataframe: dd.DataFrame,
+        dask_client: t.Optional[Client] = None,
```

Comment:
what happens if it's None, should it default to the LocalCluster? Ideally the default here would follow the one in the ComponentSpec

Reply:
I wanted to avoid failing pipelines if we merge this one. 
We will call `dd.compute(*write_tasks, scheduler=None)` what will have either tries to load the global client (if available) or using the `default client` our existing approach.
We can revisit it again when we are fixing the oom issue.

Reply:
ok that's clear, thanks!